package.json

```diff
@@ -3,10 +3,10 @@
   "version": "1.0.0",
   "description": "",
   "main": "index.js",
-  "type": "module",
   "scripts": {
-    "start": "pm2 start index.js --name ngmp-network-app",
-    "stop": "pm2 stop ngmp-network-app"
+    "start": "pm2 start src/module_6/index.ts --name ngmp-network-app",
+    "stop": "pm2 stop ngmp-network-app",
+    "dev": "tsx src/module_6/index.ts"
```

Comment:
I couldnt run the application since it is requesting an interpreter to run typescript in pm2, I recommend you to compile the entire project using `tsc` and use the generated javascript file to run it in pm2.

Reply:
Any suggestion how to do it?

I wasn't able to do this properly and the whole application was getting errors for getting tsc to run just for the specific folder

Reply:
Try using `rootDir` in your tsconfig file, if you have errors DM me.